src/Pelagos/Util/DOIutil.php

```diff
@@ -199,7 +139,7 @@ public function createDOI(
             }
         }
 
-        return true;
+        return $doi;
```

Comment:
The caller function createDoi() previous expected TRUE as return, however it doesn't appear any caller was checking for this. It's good to check if the function is still compatible.

Reply:
I did check in the consumer to make sure I wasn't breaking anything. I'll take a second careful look just to be sure.